### DISCUSSION TITLE
Step 4 sentence seems not right for the time

### DISCUSSION THREAD

**Discussion by Puckoland** [skills/.github#744] [Communicate using Markdown]
In *Step 4*, there is this part in the `README.md`:

> ## Step 4: Make a task list
> ...
> ### :keyboard: Activity: Add a task list
> GitHub Actions went ahead and made a branch for you. So you'll need to add to the file we've created in the branch, and we will check your work as you work through this course!

Why is this sentence here? It seems like it should be in the *Step 1*. That is when the branch was created. And it is not like it stayed there since *Step 1*, it was added in the commit Puckoland/skills-communicate-using-markdown@a5c9971 "Update to 4 in STEP and README.md". Or am I missing something?

**Comment by aflacake** [comment]
Yes, I also had a similar problem. Issue: [problem step 3](https://github.com/skills/communicate-using-markdown/issues/251).

**Comment by Laura003-t** [comment]
It should definitely be a mistake, just ignored, I continued my work without paying attention to that and I've completed the course
